modules/ROOT/pages/design-create-publish-api-specs.adoc

```diff
@@ -25,14 +29,14 @@ With the visual editor, you can:
 
 == Documenting
 
-As you add descriptions in your specifications and fragments, you can view them in auto-generated, interactive documentation in the right-hand pane of the text editor.
+As you add descriptions in your specifications and fragments, you can view them in auto-generated, interactive documentation in the right pane of the text editor.
 
 == Testing
 
-You can use the mocking service to:
+You can use the mocking service for RAML or OAS APIs to:
```

Comment:
Have we introduced the mocking service? Does your audience already understand how it works?